release-notes/5.0/preview/api-diff/rc1/.Net/5.0-rc1.md

```diff
@@ -0,0 +1,28 @@
+# API Difference net5.0-preview8 vs net5.0-rc1
+
+API listing follows standard diff formatting. Lines preceded by a '+' are
+additions and a '-' indicates removal.
+
+* [System](5.0-rc1_System.md)
+* [System.Buffers](5.0-rc1_System.Buffers.md)
+* [System.ComponentModel](5.0-rc1_System.ComponentModel.md)
+* [System.Data.Common](5.0-rc1_System.Data.Common.md)
+* [System.Diagnostics](5.0-rc1_System.Diagnostics.md)
+* [System.Globalization](5.0-rc1_System.Globalization.md)
+* [System.IO.IsolatedStorage](5.0-rc1_System.IO.IsolatedStorage.md)
+* [System.IO.Pipelines](5.0-rc1_System.IO.Pipelines.md)
```

Comment:
I suggest removing this.  The APIs didn't change, they were just temporarily moved to be part of netcoreapp rather than separate, and then they were subsequently moved out again.